### PR TITLE
Add html and css

### PR DESCRIPTION
**Pull request by ChrispaixK**
## "Hello Microverse" first PR

In this PR I :
* Created an HTML file that includes a header with the text "Hello Microverse!".
* Created a CSS file that includes one class with styles for my header

### CODE REVIEW

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ChrispaixK  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ GitHub flow was set up correctly


## Required Changes ♻️
- [ ]  Please make sure your Pull request description explains all the features you added in this project, preferably in bullet points. You can tell exactly which files were added and what feature are they used to implement.
- [ ]  For your next project, please ensure all commit messages have their first letter capitalized.
- [ ] For your next project, please ensure that everything should be committed in the Feature branch, not the main one. Right now I see that you added the ReadMe and Linters in the main branch so I don't see it in the Files changed for this PR/Branch.
- [ ] Also, kindly esure you add your name in the License

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
## hello @zhadier 
>I made the changes you asked me to make and took note of best practices for next time.
  Thank you!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+test.md
+.hintrc
+node_modules/
+package-lock.json
+package.json
+.stylelintrc.json
```
Comment (Kossi-stack) [inline .gitignore]: - [ ] [OPTIONAL] Kindly remove all files and leave test.md & node_modules in the gitignore

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @ChrispaixK ,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] [OPTIONAL] Kindly edit the file in the Github/workflows to linters.yml
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**